js/bitmex.js

```diff
@@ -2202,12 +2202,17 @@ module.exports = class bitmex extends Exchange {
         } else if (market['quote'] === 'USDT') {
             resultValue = Precise.stringMul (value, '0.000001');
         } else {
-            const currency = this.currency (market['quote']);
+            let currency = undefined;
+            const quote = market['quote'];
+            if (quote !== undefined) {
+                currency = this.currency (market['quote']);
+            }
             if (currency !== undefined) {
                 resultValue = Precise.stringMul (value, this.numberToString (currency['precision']));
+                return parseFloat (resultValue);
             }
         }
-        return parseFloat (resultValue);
+        return undefined;
```

Comment:
@carlosmiei Changing this line introduces another bug – we're now returning undefined in all other if-else cases, which seems wrong to me. Lmk your thoughts.

Reply:
@kroitor yes my bad, I was trying to avoid `parseFloat(undefined)` that would return `nan` without an extra check but ended up messing it, it should be ok now, thanks!